src/main.cpp

```diff
@@ -3,14 +3,89 @@
 //  Game of Life
 //
 
+#include <SFML/Graphics.hpp>
 #include <cmath>
 #include <iostream>
+#include "Grid.hpp"
 
 
 namespace {
 
+const int SCREEN_WIDTH = 1420;
+const int SCREEN_HEIGHT = 1000;
```

Comment:
Consider using unsigned types for values that should never be negative. Also consider using `sf::Vector2` instead of storing width and height separately.